tests/Domain/DonationRepositoryMatchFundsAllocationTest.php

```diff
@@ -132,7 +131,7 @@ public function testItAllocates1From1For1(): void
 
 
     /**
-     * @return list<array{0:string, 1: string, 2: string, 3: string, 4: string, 5: string}>
+     * @psalm-return list<array{0:string, 1: string, 2: numeric-string, 3: string, 4: string, 5: string}>
```

Comment:
No objection, but out of curiosity why change from @return to @psalm-return? Would you prefer if I use the vendor prefix in future?

Reply:
@bdsl I get an IDE or plugin notice about unknown types when I use `numeric-string` without a prefix, so this seemed like the simplest way to avoid adding dev noise of that sort. I don't have a strong preference for doing it everywhere.

Reply:
@NoelLH Fair enough. That might be fixed by installing this plugin - it's bundled with PHPStorm, not sure if it comes bundled with IntelliJ https://plugins.jetbrains.com/plugin/15183-psalm-support